client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -199,6 +204,16 @@ const DataFilesCopyModal = React.memo(() => {
           </div>
         </div>
       </ModalBody>
+      <div className="d-flex">
+        <ModalFooter>
+          {modalParams.scheme === 'public' && (
+            <SectionMessage type="warning">
+              Files copied to Public Data will be avaliable to general public.{' '}
+              <b>This action cannot be reversed.</b>
+            </SectionMessage>
+          )}
+        </ModalFooter>
+      </div>
```

Comment:
As it is here, if no `SectionMessage` is rendered, then the `ModalFooter` is still rendered with a line bordering the bottom of the modal. Instead, let's conditionally render this whole block. Can you move the conditional up?

Also, wrapping `ModalFooter` with `d-flex` seems to mess with the bottom border (see image). 

<img width="1118" alt="Screen Shot 2021-11-10 at 2 34 30 PM" src="https://user-images.githubusercontent.com/20326896/141190637-b4c3bf85-9477-40cc-9227-7bd98c56a3d0.png">

You will want to make `ModalFooter` the parent element of this highlighted footer code. After doing so, you may find that `d-flex` needs to be moved or replaced with a custom style to get the left-alignment of the `SectionMessage` right

Reply:
@rstijerina Do we want the `type="warning"` to be in the `ModalFooter`? Because when that is done the code breaks
<img width="889" alt="Screen Shot 2021-11-10 at 4 38 49 PM" src="https://user-images.githubusercontent.com/76450787/141205301-72840b84-5694-45a0-a5bc-d47563b7771a.png">
.

Reply:
Hmm, no, the `type` should remain with the `SectionMessage`

Reply:
<img width="1166" alt="Screen Shot 2021-11-11 at 11 17 52 AM" src="https://user-images.githubusercontent.com/76450787/141341433-859fbe10-5477-4d18-be1c-2b26db1c6632.png">